swap_meet/vendor.py

```diff
@@ -1,2 +1,89 @@
 class Vendor:
-    pass
\ No newline at end of file
+    """The Vendor class has optional arguement inventory,
+    methods: .add(item), .remove(item), .get_by_category(category),
+    swap_items(swapping_vendor, my_item, their_item)"""
+    def __init__(self, inventory = None):
+        #if an inventory list is given, assign it to the attribute
+        if inventory:
+            self.inventory = inventory
+        #if no inventory list is given, initialize an empty list for inventory.
+        else:
+            self.inventory = []
+
+    def add(self, item):
+        """appends the item to the inventory list and returns the item"""
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        """removes an item from the inventory list"""
+        #if item is in inventory, remove it
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
```

Comment:
The else isn't necessary! Since the if block has a return statement, we know that if we move past the if it must be because we're in what happens in else. The return statement can stand on its own outside of the else.